OpenRA.Game/Network/OrderManager.cs

```diff
@@ -131,8 +131,12 @@ public void TickImmediate()
 						return;
 
 					var frame = BitConverter.ToInt32(packet, 0);
-					if (packet.Length == 5 && packet[4] == (byte)OrderType.Disconnect)
-						pendingPackets.Remove(clientId);
+					if (packet.Length == 20 && packet[4] == (byte)OrderType.Disconnect)
```

Comment:
Can we define a const `Order.DisconnectOrderLength = 16` like we do for `SyncHashOrderLength` and then check it here as `packet.Length == Order.DisconnectOrderLength + 4`?